src/notify/notify.c

```diff
@@ -53,8 +59,10 @@ static int help(void) {
                "     --status=TEXT     Set status text\n"
                "     --booted          Check if the system was booted up with systemd\n"
                "     --no-block        Do not wait until operation finished\n"
+               "     --exec            Execute command line separated by ';' once done\n"
```

Comment:
please add a test for this, maybe to `TEST-20-MAINPIDGAMES` which already call systemd-notify a lot

Reply:
never mind, seen now